### ISSUE TITLE
test_threads fails with no error if there is no bbi.yaml

### ISSUE THREAD

**Issue by barrettk** [metrumresearchgroup/bbr#495]
This is possibly a somewhat larger issue. If `.wait = FALSE`, bbr will not error out saying there is no `bbi.yaml` file. In my opinion we would want this to error out in both circumstances so users aren't waiting for something to happen (see below).

![image](https://user-images.githubusercontent.com/18685116/173414747-4af67ed6-9a24-4652-b99b-262317972c03.png)

![image](https://user-images.githubusercontent.com/18685116/173414802-14734e0c-b21a-4fee-a34d-a682d30fc036.png)

For the purposes of fixing this issue in test_threads, I can simply check for the yaml file before submitting

**Comment by seth127** [comment]
Good catch @barrettk . I see you put in a targeted fix in #496 but I think you're right that this might merit a more general fix.

I didn't look too hard, but I think `submit_model()` checks for this [here](https://github.com/metrumresearchgroup/bbr/blob/develop/R/submit-model.R#L98-L109) before calling out to `bbi`. However, `submit_models()` (_plural_), which is what's called by `test_threads()` looks like it _doesn't_ check for it, unless the user passes a custom config path: look at the code [here](https://github.com/metrumresearchgroup/bbr/blob/develop/R/submit-models.R#L125-L131) and notice that the `else` block is missing.

I would interested in seeing if adding in that `else` block to `submit_models()` fixes the issue (without the fix in #496). Either way, we probably want to add that handling to `submit_models()` I would think.

**Comment by barrettk** [comment]
@seth Interestingly, it doesnt seem to fix the issue. When running `submit_models(.mods, .wait = FALSE)`, we get the appropriate error now:
```r
> submit_models(.mods, .wait = FALSE)
 Error: Looking for default configuration file at /data/Projects/package_dev/bbr/inst/model/nonmem/test-test-threads-models/bbi.yaml: no such file or directory 
```

However, test_threads doesnt seem to pass that error forward:
![image](https://user-images.githubusercontent.com/18685116/174112227-a3af3d81-c573-44a4-a1f4-da102e86d1b5.png)

Im not sure if we necessarily need to keep the test_threads patch I made, but will look into whether it can be removed, as it shouldnt be necessary
